kit/ChildSession.cpp

```diff
@@ -733,9 +733,9 @@ bool ChildSession::sendFontRendering(const char* /*buffer*/, int /*length*/, con
 
     ptrFont = getLOKitDocument()->renderFont(decodedFont.c_str(), decodedChar.c_str(), &width, &height);
 
-    const auto duration = std::chrono::system_clock::now() - start;
-    const auto elapsed = std::chrono::duration_cast<std::chrono::microseconds>(duration).count();
-    LOG_TRC("renderFont [" << font << "] rendered in " << elapsed << "ms");
```

Comment:
Apparently we were reporting microseconds as milliseconds, giving 1000x longer rendering time.